src/operation-node/unique-constraint-node.ts

```diff
@@ -7,8 +7,14 @@ export interface UniqueConstraintNode extends OperationNode {
   readonly kind: 'UniqueConstraintNode'
   readonly columns: ReadonlyArray<ColumnNode>
   readonly name?: IdentifierNode
+  readonly nullsNotDistinct?: boolean
 }
 
+export type UniqueConstraintNodeProps = Omit<
+  Partial<UniqueConstraintNode>,
+  'kind'
+>
```

Comment:
is this used outside of `cloneWith`? if not, remove.

Reply:
Same, we need this change